src/scm/gitlab.js

```diff
@@ -236,23 +252,27 @@ class Gitlab {
     if (!branch || branch === '') return;
     return new Promise((resolve, reject) => {
         return $.ajax({
-          url: `${this.baseUrl}/projects/${context.repo.id}/repository/branches`,
-          headers: this.tokenHeader,
-          method: 'POST',
-          crossDomain: true,
-          dataType: 'json',
-          contentType: 'application/json',
-          data: JSON.stringify(payload)
-        })
+            url: `${this.baseUrl}/projects/${context.repo.id}/repository/branches`,
+            headers: this.tokenHeader,
+            method: 'POST',
+            crossDomain: true,
+            dataType: 'json',
+            contentType: 'application/json',
+            data: JSON.stringify(payload)
+          })
           .then(resolve)
           .fail(reject);
       })
-        .then(response => {
-          context.branch = branch;
-          Object.assign(context.bindBranch, { [context.id] : branch });
-          chrome.storage.sync.set({ bindBranch: context.bindBranch });
-          return branch;
-        })
+      .then(response => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/gitlab.js#L266"></span>‘response’ is defined but never used.</td>
    </tr>
  </tbody>
</table>